src/utils.js

```diff
@@ -20,6 +20,33 @@ const utils = {
         return out;
     },
 
+    xorArrayBuffers: function (x, y) {
+        let xIntArray = new Uint8Array(x);
+        let yIntArray = new Uint8Array(y);
+        let zIntArray = new Uint8Array(x.byteLength);
+        for (let i = 0; i < x.byteLength; i++) {
+             zIntArray[i] = xIntArray[i] ^ yIntArray[i];
+        }
+        return zIntArray.buffer;
+    },
+
+    arrayBufToBase64: function ( buffer ) {
+        /* This function is due to mobz (https://stackoverflow.com/users/1234628/mobz)
+        *  and Emmanuel (https://stackoverflow.com/users/288564/emmanuel)
+        */
+        var binary = '';
+        var bytes = new Uint8Array( buffer );
+        var len = bytes.byteLength;
+        for (var i = 0; i < len; i++) {
```

Comment:
Please replace the `var` statements with `let` or `const`.

Reply:
The rest of these look fine. I'll go ahead and fix these up.